vpr/src/route/rr_graph_generation/rr_graph.cpp

```diff
@@ -1667,6 +1670,13 @@ static std::function<void(t_chan_width*)> alloc_and_load_rr_graph(RRGraphBuilder
         }
     }
 
+    if (!grid.get_horizontal_interposer_cuts().empty() || !grid.get_vertical_interposer_cuts().empty()) {
```

Comment:
I think it would be better if you add a comment explaining what is happening here. It would be helpful to mention why the extra edges are created and why the following step (removing them) is necessary.